test/utils/Addresses.sol

```diff
@@ -4,6 +4,7 @@ pragma solidity 0.7.6;
 import "forge-std/Test.sol";
 
 address constant ETH_ADDRESS = 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
+address constant ZERO_ADDRESS = address(0);
```

Comment:
Is zero address a new convention to refer to ETH? Why do we need it?

Reply:
Actually it's not new and it's aligned with `AMMWrapper` contract.
https://github.com/consenlabs/tokenlon-contracts/blob/v5/contracts/AMMWrapper.sol#L161